cityscrape-config.sh

```diff
@@ -0,0 +1,16 @@
+# Variables for the cityscrape
+
+# Postgresql Database Settings
+# make sure you have enabled PostGIS
+export DATABASE_HOST='localhost'
+export DATABASE_NAME='city'
+export DATABASE_USER='postgres'
+export DATABASE_PASSWORD=''
```

Comment:
calling out sensitive variables with something like [redacted] might make it more apparent that this was intentionally left out.